### PR TITLE
added 'album' foreign key to models

### PR DESCRIPTION
**Pull request by C-Anirudh**
Fixes: #13 
Added 'album' as foreign key to three models:
- activity.Project
- activity.Event
- blog.Post

Added unique=True attribute to the `title` field of Album model in gallery app.

### CODE REVIEW

File: gallery/models.py [hunk 1]
```diff
@@ -25,7 +25,7 @@ class Meta:
 
 
 class Album(models.Model) :
-    title = models.CharField(max_length=200)
+    title = models.CharField(max_length=200, unique=True)
```
Comment (aswinshenoy) [inline gallery/models.py]: Why do you need to make it unique? there is already an ID which will act as the primary key.

**Review by aswinshenoy — APPROVED** [review]
clean, and does the job.

File: activity/models.py [hunk 2]
```diff
@@ -23,6 +24,7 @@ def get_poster_path(instance, filename):
     topics = models.ManyToManyField(Skill, related_name='ProjectTopics', blank=True)
     detail = RichTextField(verbose_name='Details', max_length=10000, null=True)
     links = models.ManyToManyField(Portal, related_name='ProjectLinks', through='ProjectLink')
+    album = models.ForeignKey(Album, on_delete=models.CASCADE, null=True, blank=True, to_field='title')
```
Comment (aswinshenoy) [inline activity/models.py]: ```to_field``` attribute is not required, It should automatically point to the primary key (ID of the album). At the same time, we shouldn't be using publicly visible primary key like title either.

**Comment by aswinshenoy** [comment]
By default, Django uses the primary key of the related object. If you reference a different field, that field must have unique=True. 

Here, we follow using the primary key of the object (ID) to reference an object in any Foreign Key relationship throughout our project.

**Comment by C-Anirudh** [comment]
I thought it would be easier to select albums in the blog and activity models by name rather than their ID.

**Comment by aswinshenoy** [comment]
It would be still callable by name, just check it out.

**Pull request merged**